test/test_store.py

```diff
@@ -65,3 +65,14 @@ def test_data_opener_params_schema(self):
         self.assertIn('spatial_res', schema.properties)
         self.assertIn('bbox', schema.properties)
         self.assertIn('crs', schema.properties)
+
+    def test_get_type_specifiers(self):
```

Comment:
Please add @unittest.skipUnless(HAS_SH_CREDENTIALS, REQUIRE_SH_CREDENTIALS) to both tests. Also, I found that test_data_opener_params_schema fails when the credentials are not set, as more parameters are required.

Reply:
Right.